msi/GitProduct.wxs

```diff
@@ -158,6 +167,11 @@
       <Directory Id='ProgramFiles$(var.SixtyFourBit)Folder' Name='PFiles'>
         <Directory Id='INSTALLFOLDER' Name='Git'>
           <Directory Id='BinFolder' Name='bin' />
+        <?if $(var.SixtyFourBit)=64 ?>
+          <Directory Id='MingwFolder' Name='mingw64' />
+        <?else?>
+          <Directory Id='MingwFolder' Name='mingw32' />
+        <?endif?>
```

Comment:
I tried to do what I suggested above, but this is not valid WiX XML. So, this conditional will need to stay. But in the future, I will keep in mind that whenever possible, conditional statements with respect to generating WiX XML should, as much as possible, be limited to being generated from `release.sh` rather than via WiX itself.